openedx/core/lib/properties.py

```diff
@@ -0,0 +1,64 @@
+"""
+Generally useful property subclasses.
+"""
+import json
+
+
+class JSONProperty(property):
```

Comment:
You can move this to openedx/core/lib/api where pre-existing Fields and Serializers exist.